client/landing/gutenboarding/components/domain-picker/style.scss

```diff
@@ -15,6 +15,19 @@
 	}
 }
 
+.domain-picker__panel-row {
+	position: relative;
+}
+
+.domain-picker__close-button {
+	// Close button has to appear after search input
+	// as @wordpress/popover finds and focuses on
+	// the first found focusable element in the popover content.
+	position: absolute;
+	top: 0;
+	right: 0;
+}
+
```

Comment:
I'm not sure the find & focus thing justifies altering the structure in this case. Can we programatically trigger focus using a ref like we do in `client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx`? 🤔

Reply:
> I'm not sure the find & focus thing justifies altering the structure in this case. Can we programatically trigger focus using a ref like we do in client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx? 🤔

Indeed I was taking the easy way out by not having to programmatically trigger a focus. I guess there's no avoiding the imminent now!!!

> Another option would be to add tabIndex={-1} to the CloseButton which currently does exactly the same thing as Confirm button. And we can also add a comment to explain this context.

The close button should be tabbable no?

Reply:
It's better to not handle focus on your own and let the component do it as it has accessibility considerations.